src/runners/modal-runner.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WTEGModal from '../../components/modal/wtegModel/wtegModal.react';
+{
+  const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent),
+    detailsPage = document.querySelector('.details-page');
+
+  if (!isMobile) {
+    if (detailsPage.contains(document.getElementById('wteg-image-wrapper'))) {
+      if (document.getElementById('wteg-image-wrapper')) {
```

Comment:
The two tests for presence of `wteg-image-wrapper` are redundant. The one will do.